### PR TITLE
Tutorial proposal called "Create and deploy a DeFi App"

### PR DESCRIPTION
**Pull request by strykerin**
Here is the tutorial I wrote about creating and deploying a DeFi app. It is a proposal to be published in the Tutorials section.

In this tutorial we will build a DeFi Application with solidity where users can deposit an ERC20 token to the smart contract and it will mint and transfer Farm Tokens to them. The users can later withdraw their ERC20 tokens by burning their Farm Token on smart contract and the ERC20 tokens will be transferred back to them.

### CODE REVIEW

File: src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+    console.log('Balance FarmToken After TokenFarm ' + web3.utils.fromWei(balanceFarmTokenAfterFarmToken.toString()))
+
+    // End function
+    callback();
+}
+```
+
+To run this script: `truffle exec .\scripts\transferMyTokenToFarmToken.js`. You should see on your console:
+
+![output of transferMyTokenToFarmToken.js](https://cdn-images-1.medium.com/max/2000/1*MoekE2QCw7vB98u5dl7ang.png)*output of transferMyTokenToFarmToken.js*
```
Comment (wackerow) [inline src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md:354]: similar to above, consider bringing bold caption down two rows in markdown for readability

File: src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+    console.log('Balance FarmToken After TokenFarm ' + web3.utils.fromWei(balanceFarmTokenAfterFarmToken.toString()))
+
+    // End function
+    callback();
+}
+```
+
+To run this script: `truffle exec .\scripts\withdrawMyTokenFromTokenFarm.js`. As we can see on the output below, we have successfully got the MyTokens back and we have burned the FarmTokens:
+
+![output of withdrawMyTokenFromTokenFarm.js](https://cdn-images-1.medium.com/max/2000/1*jHYlTFg0NgGbhASpsRvc0w.png)*output of withdrawMyTokenFromTokenFarm.js*
```
Comment (wackerow) [inline src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md:405]: (same)

File: src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+
+## References
```
Comment (wackerow) [inline src/content/developers/tutorials/create-and-deploy-a-defi-app/index.md:407]: `{#references}`

**Review by wackerow — CHANGES_REQUESTED** [review]
@strykerin This is great! Thanks for putting this together, I enjoyed reading through it... just noted a couple small things to get it ready for the site. 👍

**Comment by samajammin** [comment]
Thanks for submitting this @strykerin! 

Great comments @wackerow - I agree with all them. I'll go ahead & merge this then add changes for the sake of speed.

**Comment by samajammin** [comment]
@all-contributors please add @strykerin for content.

**Pull request merged**